service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/EhrExtractMapperTest.java

```diff
@@ -145,6 +156,15 @@ public void When_MappingProperJsonRequestBody_Expect_ProperXmlOutput() throws IO
         assertThat(output).isEqualTo(expectedJsonToXmlContent);
     }
 
+    private static Stream<Arguments> testData() {
+        return Stream.of(
+            Arguments.of(JSON_INPUT_FILE, EXPECTED_XML_TO_JSON_FILE),
+            Arguments.of(FHIR_BUNDLE_WITHOUT_EFFECTIVE_TIME, EXPECTED_XML_WITHOUT_EFFECTIVE_TIME),
+            Arguments.of(FHIR_BUNDLE_WITHOUT_HIGH_EFFECTIVE_TIME, EXPECTED_XML_WITHOUT_HIGH_EFFECTIVE_TIME),
```

Comment:
Should we also cover `EXPECTED_XML_WITHOUT_LOW_EFFECTIVE_TIME`?

Reply:
I couldn't find a resource that is mapped to only high, all of them have low required or setting to nullFlavor if low does not exist. And I am running the tests through all the mappers, because I want to check it agains consultation and non consultation. For non consultation we are ignoring high at all.

Reply:
So in fact it it not possible case we have only high at the moment. Unless I checked it wrong, but I spent quite a lot of time analysing all the mappings.

Reply:
Ok, that's fine - thanks for checking